openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -75,11 +79,11 @@ class _DispatchingViewTestCase(TestCase):
 
     Subclasses need to define self.url.
     """
-    dop_adapter = adapters.DOPAdapter()
-    dot_adapter = adapters.DOTAdapter()
 
     def setUp(self):
         super(_DispatchingViewTestCase, self).setUp()
+        self.dop_adapter = adapters.DOPAdapter()
+        self.dot_adapter = adapters.DOTAdapter()
```

Comment:
The former class attributes in these classes could probably also go into `setUpClass`